src/NotificationTargetCommonITILObject.php

```diff
@@ -2073,7 +2107,11 @@ public function getTags()
             $objettype . '.numberoflinkedtickets' => _x('quantity', 'Number of linked tickets'),
             $objettype . '.numberoflinkedchanges' => _x('quantity', 'Number of linked changes'),
             $objettype . '.numberoflinkedproblems' => _x('quantity', 'Number of linked problems'),
-
+            $objettype . '.autobump.bumpcounter' => __('Number of bumps'),
+            $objettype . '.autobump.bumpremaining' => __('Number of remaining bumps'),
+            $objettype . '.autobump.bumptotal'  => __('Total number of bumps'),
+            $objettype . '.autobump.deadline'   => __('Deadline'),
+            $objettype . '.autobump.bumptext'   => __('Bump text'),
```

Comment:
I would add a little more details to make the fields less ambiguous:
```suggestion
            $objettype . '.autobump.bumpcounter' => __('Number of sent bumps'),
            $objettype . '.autobump.bumpremaining' => __('Number of remaining bumps'),
            $objettype . '.autobump.bumptotal'  => __('Total number of bumps before auto resolution'),
            $objettype . '.autobump.deadline'   => __('Auto resolution deadline'),
            $objettype . '.autobump.bumptext'   => __('Bump text'),
```